platformio.ini

```diff
@@ -33,8 +33,8 @@ lib_deps =
 bt_deps =
 	BluetoothSerial
 wifi_deps =
-	arduinoWebSockets=https://github.com/MitchBradley/arduinoWebSockets#7793f9e
-	WiFi=https://github.com/MitchBradley/WiFi#e63583f
+	arduinoWebSockets=https://github.com/MitchBradley/arduinoWebSockets#canSend
+	WiFi=https://github.com/MitchBradley/WiFi#canWrite
         ; If we include the following explicit dependencies when we get the
         ; Arduino framework code from github, platformio picks up different
         ; and incompatible versions of these libraries from who knows where, 
```

Comment:
you probably want to update line 53 to version 2.0.15 too

    framework-arduinoespressif32@ https://github.com/espressif/arduino-esp32.git#2.0.15

Reply:
Indeed, thanks